### DISCUSSION TITLE
Lost match brackets action for RPG

### DISCUSSION THREAD

**Discussion by m-tyler** [codefori/vscode-ibmi#727] [General]
After updating to VS Code 1.68.1 and then this extension 1.3.9, I no longer have the ability to match to IF/ENDIF, BEGSR/ENDSR or similar that used to be there.

Is there a change to VS Code or this extension that maybe indicate its defined another way?

**Comment by worksofliam** [comment]
This is related to ibmi-languages, not vscode-ibmi.
